google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/admin/v2/MockBigtableInstanceAdmin.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017, Google LLC All rights reserved.
```

Comment:
This is an auto-generated file. So are several package-info.java files and Test*.java files.